docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,11 @@ develop
     *    - Type
          - Change
 
+    *    - |improved| |logs| |metrics|
+         - Allow StreamStore table names to be marked as safe. This will make StreamStore tables appear correctly on our logs and metrics.
+           When building a StreamStore, please use `.tableNameLogSafety(TableMetadataPersistence.LogSafety.SAFE)` to mark the table name as safe.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2835>`__)
```

Comment:
We should this to the docs to where we document log safety definition semantics and the stream store definition docs.

The change looks great overall. Thanks! Interestingly, adding this does not change any gen code.